src/main/docs/guide/micronautOpenApiAnnotations/openApiRequest.adoc

```diff
@@ -0,0 +1,73 @@
+Annotation api:openapi.annotation.OpenAPIRequest[] can be used as alternative to *io.micronaut.http.annotation.RequestBean* to mark parameter as wrapped request details. It useful, when you haven't micronaut-http in dependencies. For example, when you use micronaut-openapi with spring-boot application.
```

Comment:
describe what dependency is needed on the path for this. It can't be the `openapi` module since that will pull in all the processing dependencies

Reply:
Could you be more specific about what needs to be fixed? I didn't understand what needs to be added.